pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -29,6 +29,9 @@ import (
 )
 
 // A stateful listener builder
+// Support the below intentions
+// 1. Use separate inbound capture listener(:15006) and outbound capture listener(:15001)
+// 2. The above listeners use bind_to_port sub listeners or filter chains.
```

Comment:
For master - do we have any reason to support bind_to_port on inbound ?

Reply:
The `listener_builder` is now supporting the behavior of no splitting since the commit by @rshriram  days ago.